src/tests/category.rs

```diff
@@ -38,3 +41,57 @@ fn show() {
     let json: GoodCategory = ::json(&mut response);
     assert_eq!(json.category.category, "foo".to_string());
 }
+
+fn tx(req: &Request) -> &GenericConnection { req.tx().unwrap() }
+
+#[test]
+fn update_crate() {
```

Comment:
There are a lot of tests here. Some comments to outline **what** is being tested at each assertion group would be useful.